pkgs/applications/misc/xdragon/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, pkg-config, gtk3 }:
+
+stdenv.mkDerivation rec {
+  pname = "xdragon";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "mwh";
+    repo = "dragon";
+    rev = "v${version}";
+    sha256 = "0fgzz39007fdjwq72scp0qygp2v3zc5f1xkm0sxaa8zxm25g1bra";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ gtk3 ];
+
+  installFlags = [ "PREFIX=${placeholder "out"}/bin" ];
+  postInstall = ''
+    mv $out/bin/dragon $out/bin/xdragon
+  '';
```

Comment:
if the intended behavior is for it to be named dragon, we should keep it as dragon. and maybe just add a link to it:
```
  ln -s $out/bin/dragon $out/bin/xdragon
```

if this was done to avoid the scenario of conflicts with the existing `dragon` package, then we use `lib.lowPrio` in `all-packages.nix` to make the existing dragon prefered